tests/foreman/ui/test_ansible.py

```diff
@@ -115,3 +119,65 @@ def test_positive_create_variable_with_overrides(target_sat):
             }
         )
         assert session.ansiblevariables.search(key)[0]['Name'] == key
+
+
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_config_report_ansible(session, request, target_sat, module_org, rhel_contenthost):
+    """Test Config Report generation with Ansible Jobs.
+
+    :id: 118e25e5-409e-44ba-b908-217da9722576
+
+    :steps:
+        1. Register a content host with satellite
+        2. Import a role into satellite
+        3. Assign that role to a host
+        4. Assert that the role was assigned to the host successfully
+        5. Run the Ansible playbook associated with that role
+        6. Check if the report is created successfully
+
+    :expectedresults:
+        1. Host should be assigned the proper role.
+        2. Job report should be created.
+
+    :BZ: 2155814
+    """
+    SELECTED_ROLE = 'RedHatInsights.insights-client'
+    if rhel_contenthost.os_version.major == 7:
```

Comment:
I know we're skipping el6, but just to be explicit  
```suggestion
    if rhel_contenthost.os_version.major <= 7:
```